src/components/members/ToggleSkillIcon.tsx

```diff
@@ -0,0 +1,119 @@
+'use client';
+
+import { CheckIcon, XIcon } from 'lucide-react';
+import { useTranslations } from 'next-intl';
+import { toast } from 'sonner';
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogActionDestructive,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from '@/components/ui/AlertDialog';
+import { Button } from '@/components/ui/Button';
+import { Spinner } from '@/components/ui/Spinner';
+import { api } from '@/lib/api/client';
+import { useRouter } from '@/lib/locale/navigation';
+import type { RouterOutput } from '@/server/api';
+
+function ToggleSkillIcon({
+  user,
+  skill,
+  hasSkill,
+  editable,
+}: {
+  user: NonNullable<RouterOutput['users']['fetchUser']>;
+  skill: RouterOutput['skills']['fetchAllSkills'][number];
+  hasSkill: boolean;
+  editable: boolean;
+}) {
+  const t = useTranslations('members.skillManagement');
+  const tSkills = useTranslations('skills');
+  const router = useRouter();
+  const addSkill = api.skills.addSkillToUser.useMutation({
+    onSuccess: router.refresh,
+  });
+  const removeSkill = api.skills.removeSkillFromUser.useMutation({
+    onSuccess: router.refresh,
+  });
+
+  const trigger = hasSkill ? (
+    <CheckIcon className='h-4 w-4 text-green-500' />
+  ) : (
+    <XIcon className='h-4 w-4 text-red-500' />
+  );
+
+  if (!editable) return trigger;
+
+  return (
+    <AlertDialog>
+      <AlertDialogTrigger asChild>
+        <Button variant='none' size='none' className='block'>
```

Comment:
```suggestion
        <Button
          variant='outline'
          size='xs-icon'
          className='block [&>*]:mx-auto'
        >
```
Makes it obvious that they're interactible.